pkg/logging/scalyr/scalyr_test.go

```diff
@@ -12,23 +12,33 @@ import (
 	"github.com/fastly/go-fastly/fastly"
```

Comment:
Scalyr non-exported tests previously did not accurately follow the testing pattern of the other logging endpoints.